### PR TITLE
[15.0][MIG] database_cleanup

### PR DESCRIPTION
**Pull request by damdam-s**
based on #1990

### CODE REVIEW

**Comment by gurneyalex** [comment]
https://github.com/damdam-s/server-tools/pull/1 fixes an issue I had with this version.

**Comment by JuanyDForgeflow** [comment]
@damdam-s  Can you check the tests?

**Comment by leemannd** [comment]
Original 14.0 PR has been superseed by -> database_cleanup https://github.com/OCA/server-tools/pull/2390

File: database_cleanup/views/purge_menus.xml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        </field>
+    </record>
+
+    <record id="action_purge_menus" model="ir.actions.server">
+        <field name="name">Purge menus</field>
+        <field name="type">ir.actions.server</field>
+        <field name="state">code</field>
+        <field name="model_id" ref="database_cleanup.model_cleanup_purge_wizard_menu" />
+        <field name="code">
+            action = env.get('cleanup.purge.wizard').get_wizard_action()
```
Comment (AungKoKoLin1997) [inline database_cleanup/views/purge_menus.xml:18]:
```suggestion
            action = env.get('cleanup.purge.wizard.menu').get_wizard_action()
```

Comment (AungKoKoLin1997) [reply database_cleanup/views/purge_menus.xml:18]: I tested in local environment and get error "You are not allowed to create 'Purge stuff' (cleanup.purge.wizard) records." when click "Purge absolete menu entries". I think you should use the model of current menu.

**Review by AungKoKoLin1997 — CHANGES_REQUESTED** [review]

**Comment by leemannd** [comment]
Hello @damdam-s the 14.0 has been merged from another PR that superseeds the one you have been using. Do you have time to take a look at it?

**Comment by pedrobaeza** [comment]
Superseded by #2547

**Pull request closed without merging**